src/main/java/stepsdesign/ControlCenterSteps.java

```diff
@@ -0,0 +1,205 @@
+import org.slf4j.LoggerFactory;
+
+/**
+ * Represents CdfControlCenterSteps
+ */
+public class ControlCenterSteps implements CdfHelper {
+
+  private static final Logger logger = LoggerFactory.getLogger(ControlCenterSteps.class);
+
+  @Then("Click on the Hamburger bar on the left panel")
+  public static void clickOnTheHamburgerIcon() {
+    CdfControlCenterActions.clickOnHamburgerMenu();
+  }
+
+  @Then("Click on Control Center link from the hamburger menu")
+  public void openControlCenterPage() {
+    CdfControlCenterActions.clickOnControlCenterTab();
+  }
+
+  @Then("Verify that the user is navigated to control center page successfully")
+  public void navigateControlCenterPage() {
+    CdfControlCenterActions.navigateToControlCenter();
+  }
+
+  @Then("Verify user is able to click on the create button to create a pipeline successfully")
+  public void createButtonControlCenterPage() {
+    CdfControlCenterActions.clickOnCreateButtonControlCenterPage();
+  }
+
+  @Then("Verify the pipeline created successfully is present in control center page")
+  public void pipelinePresentControlCenterPage() {
+    CdfControlCenterActions.pipelinePresentControlCenterPage();
+  }
+
+  @Then("Click on the delete icon of the created pipeline and pipeline should get deleted successfully")
+  public void deleteControlCenterPipeline() {
+    CdfControlCenterActions.deletePipelineControlCenter();
+  }
+
+  @Then("Verify the deleted pipeline is not present in the control center page")
+  public void pipelineDeletedIsNotPresent() {
+    CdfControlCenterActions.pipelineDeletedIsNotPresent();
+  }
+
+  @Then("Verify the user is able to set the preferences for the created pipeline in the control center page")
+  public void setPreferencesForPipeline() {
+    CdfControlCenterActions.clickOnSetPreferencesIcon(); }
+
+  @Then("Verify the user is able to enter the value in the key input field {string}")
+  public void setPreferencesKeyInputValue(String value) {
+    CdfControlCenterActions.enterKeyInputValue(value); }
+
+  @Then("Verify the user is able to enter the value of the key in the value input field {string}")
+  public void setPreferencesValueInput(String value) {
+    CdfControlCenterActions.enterValueInput(value);
+  }
+
+  @Then("Verify user is able to click on save and close button of set preferences")
+  public void clickSaveAndCloseButton() {
+    CdfControlCenterActions.clickOnSaveAndCloseButton(); }
+
+  @Then("Verify user is able to click on the data pipeline added in the control center page")
+  public void clickOnDataPipelineTab() {
+    CdfControlCenterActions.clickOnDataPipelineTab(); }
+
+  @Then("Verify user is able to click on the delete icon of preferences to delete the added preferences successfully")
+  public void clickOnDeleteIconSetPreferences() {
+    CdfControlCenterActions.clickOnDeleteIconSetPreferences(); }
+
+  @Then("Verify user is able to click on the dataset entity icon to see the details page of the dataset")
```

Comment:
Verify the user is able to click dataset entity icon to navigate to details page

Reply:
changed